monitoring/count-items-cronjob/alerts.yaml

```diff
@@ -0,0 +1,66 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: count_items_cronjob
+    type: constant
+    value: artesca-data-ops-count-items
+  - name: count_items_job_duration_threshold
+    type: config
+    value: 3600
+
+groups:
+  - name: count-items-cronjob/alerts.rules
+    rules:
+      - alert: CountItemsJobTakingTooLong
+        Expr: |
+          (
+            time() - 
+            label_replace(
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${count_items_cronjob}"}
+              )
+              BY (job_name, owner_name, namespace)
+              == ON(owner_name) GROUP_LEFT()
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${count_items_cronjob}"}
+              )
+              BY (owner_name),
+              "cronjob", "$1", "owner_name", "(.+)")
+          ) > ${count_items_job_duration_threshold}
+        Labels:
+          severity: critical
+        Annotations:
+          description: "Job {{$labels.job_name}} is taking more than 1h to complete"
```

Comment:
nit: for the user to better understands the impacts of this alert it may be interesting to precise them in this description. E.g `Job {{$labels.job_name}} is taking more than 1h to complete. This means that consumption information computation will be delayed and the reported metrics might not be accurate anymore.`